### PR TITLE
MS-931 Certain characters and text not allowed

### PR DESCRIPTION
**Pull request by jjrobothamho**
- Added Error message for server error when invalid character is used

### CODE REVIEW

File: apps/nrm/views/content/text-area-guidance.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<span
+id="with-hint-info"
+class="govuk-hint govuk-character-count__message"
+aria-live="polite"
+>
```
Comment (sulthan-ahmed) [inline apps/nrm/views/content/text-area-guidance.md]: did you mean to have the `>` it's messing up the below

File: apps/nrm/fields/index.js [hunk 2]
```diff
@@ -47,21 +47,45 @@ module.exports = {
...
+    }].concat(ukLocalAuthorities),
   },
   'local-authority-contacted-about-child-local-authority-phone': {
     mixin: 'input-text',
-    validate: ['required']
+    validate: ['required'],
+    attributes: [
+      {
+        attribute: 'maxlength',
+        value: 16000
```
Comment (sulthan-ahmed) [inline apps/nrm/fields/index.js]: It needs to be 1550, the jira ticket has changed

**Comment by sulthan-ahmed** [comment]
I think it is now 1550, might be good to check the jira ticket

Also, didn’t understand the >

**Comment by jjrobothamho** [comment]
I will be removing the word limit as that is a separate ticket and only update the error message

**Comment by sulthan-ahmed** [comment]
what do you mean by
>Added Guidance text to text area (WIP - dependant on MS-940 being merged to master so additional changes can be made)

**Pull request merged**